package.json

```diff
@@ -16,15 +16,20 @@
     "reactstrap": "^8.0.0",
     "sequelize": "^5.8.6"
   },
-  "devDependencies": {},
   "scripts": {
-    "test": "echo \"Error: no test specified\" && exit 1",
-    "start": "node server.js"
+    "start": "node server.js",
+    "heroku-postbuild": "cd pokedex && npm install && npm run build"
+
```

Comment:
"start": "node server.js",
    "heroku-postbuild": "cd pokedex && npm install && npm run build"

this is the correct one